activation/activation.go

```diff
@@ -677,7 +670,6 @@ func (b *Builder) createAtx(
 		nipostState.NumUnits,
 		nonce,
 	)
-	atx.InnerActivationTx.NodeID = atxNodeID
```

Comment:
Instead of `createAtx` creating a `types.ActivationTx` to then convert it later in `broadcast` to the wiret ype `createAtx` should only care about crating a wire type and returning that to be used by `broadcast`.